go.mongodb.org/mongo-driver/mongo/db.go

```diff
@@ -0,0 +1,52 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
```

Comment:
This needs to licensing needs to be resolved similarly to https://github.com/open-telemetry/opentelemetry-go-contrib/pull/15#pullrequestreview-398550926. The ask is to have DataDog sign off on this being solely licensed under `The OpenTelemetry Authors`.